builtin/common/filterlist.lua

```diff
@@ -290,8 +290,15 @@ function sort_mod_list(self)
 		if a.type ~= b.type or a.loc ~= b.loc then
```

Comment:
There's a comment above here that should be updated to say `-- Show game and world mods at bottom`

Reply:
Oops. Missed that. Thanks.

Also see #10422 where we discuss potentially removing the ".../game/mods" feature from MT.
I'd still like to get this change in. Can always remove it again.

Honestly the hardest part was go get the sorting right in this one, so that Game mods sort in the end and worldmods sort second to last.

Reply:
#10439